boardgame-ui/src/components/timer/AddTimerForm.vue

```diff
@@ -0,0 +1,298 @@
+          <b-numberinput min="0" controls-position="compact" v-model="reload_increment_seconds"/>
+        </b-field>
+      </b-field>
+
+      <b-field :label="$t('add-edit-game.board-game.label')">
+        <b-autocomplete
+          v-model="searchString"
+          :data="filteredBoardGames"
+          field="name"
+          icon="search"
+          @select="selectBoardGame"
+          name="boardGame"
+          :data-vv-as="$t('add-edit-game.board-game.label')"
+        >
+          <template slot="empty">{{$t('add-edit-game.board-game.no-result')}}</template>
+        </b-autocomplete>
+      </b-field>
+
+      <h2 class="subtitle">{{$t('add-edit-game.players.title')}}</h2>
+
+      <table class="table is-fullwidth">
+        <thead>
+        <tr>
+          <th>{{$t('add-edit-game.players.user')}}</th>
+          <th>{{$t('global.color')}}</th>
+          <th class="has-text-white">.</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr v-for="({user}, idx) in players" :key="user ? `user${user.id}` : idx">
+          <td>
+            <b-field
+              :type="{'is-danger': errors.has(`user-${idx}`)}"
+              :message="errors.first(`user-${idx}`)"
+            >
+              <user-autocomplete
+                size="is-small"
+                v-model="players[idx].user"
+                :users="allUsers"
+                :excludedIds="selectedUsersIds"
+                :name="`user-${idx}`"
+                :data-vv-as="$t('add-edit-game.players.user')"
+                v-validate="'required'"
+              />
+            </b-field>
+          </td>
+          <td>
+            <verte
+              v-model="players[idx].color"
+              picker="square"
+              model="hex"
+              :enableAlpha="false"></verte>
+          </td>
+          <td>
+            <button type="button" class="delete" @click="removePlayer(idx)"></button>
+          </td>
+        </tr>
+        <tr>
+          <td colspan="3" class="has-text-centered">
+            <button class="button is-small" type="button" @click="addPlayer()">
+              {{$t('button.add-player')}}
+            </button>
+          </td>
+        </tr>
+        </tbody>
+      </table>
+
+      <div class="buttons is-right">
+        <button class="button" type="button" @click="$emit('close')">{{$t('button.cancel')}}</button>
+        <button class="button is-primary">{{$t('button.save')}}</button>
+      </div>
+    </form>
+  </div>
+</template>
+
+<script>
+import Timer, { TimerTypes } from '@/utils/api/Timer';
+import UserAutocomplete from '@/components/form/UserAutocomplete';
+import Event from '@/utils/api/Event';
+import User from '@/utils/api/User';
+import BoardGame from '@/utils/api/BoardGame';
+import Verte from 'verte';
+import 'verte/dist/verte.css';
+
+/**
+ * emits: close, timerCreated
+ */
+export default {
+  name: 'TimerEditionPage',
```

Comment:
Same name as TimerCreationPage.vue component => makes it hard to debug
If not specified, the component name is automatically parsed from name file => unless necessary (i.e. in case of recursive component), I suggest we do not define it to avoid such situations

Reply:
👍